tests/sentry/sentry_apps/tasks/test_sentry_apps.py

```diff
@@ -415,6 +416,107 @@ def test_error_created_sends_webhook(self, safe_urlopen):
             "Sentry-Hook-Signature",
         }
 
+    def test_project_filter_no_filters_sends_webhook(self, safe_urlopen):
+        with assume_test_silo_mode_of(ServiceHookProject):
+            ServiceHookProject.objects.all().delete()
+            ServiceHook.objects.all().delete()
+
+        self.create_service_hook(
+            project_ids=[],  # no project filter
+            installation=self.install,
+            application=self.sentry_app,
+            events=["issue.created"],
+            org=self.organization,
+            actor=self.install,
```

Comment:
ditto

Reply:
the[ `create_sentry_app_installation`](https://github.com/getsentry/sentry/blob/crl/workflow-notification-slo/tests/sentry/sentry_apps/tasks/test_sentry_apps.py#L657) factory will create an installation with a `servicehook` with [no `project_ids` ](https://github.com/getsentry/sentry/blob/master/src/sentry/sentry_apps/installations.py#L145)if that's helpful?

Reply:
yes, thanks! I ended up changing this test to be closer to what I believe happens IRL when new integration is created and then webhook url + events checkboxes are updated
(using [`create_or_update_service_hooks_for_installation`](https://github.com/getsentry/sentry/blob/63462768ebd10f049c561fa4220f470adf0e3f34/src/sentry/utils/sentry_apps/service_hook_manager.py#L5))